pkg/operator/ceph/csi/controller.go

```diff
@@ -89,6 +90,11 @@ func add(ctx context.Context, mgr manager.Manager, r reconcile.Reconciler, opCon
 	}
 	logger.Infof("%s successfully started", controllerName)
 
+	err = addonsv1alpha1.AddToScheme(mgr.GetScheme())
```

Comment:
can you add a comment what exactly we are doing here

Reply:
added